### DISCUSSION TITLE
How long do teams usually take to get approved? Are teams rejected as well?

### DISCUSSION THREAD

**Discussion by ShayanShamsi** [ethz-spylab/satml-llms-ctf-issues#21] [Questions]
Hello! I was wondering how long your approval process looks like. We submitted the form 3 days ago. Since we are a team consisting of five students, could it be that our team registration is not accepted?

The team's name is "Fourth Wall" and the email address submitted with the form was `removed for privacy reasons`.

**Comment by dedeswim** [comment]
Hi, thanks a lot for your interest in our competition. All teams who recently requested credits (including yours) have just been approved. Apologies for the delay!

**Reply by ShayanShamsi** [reply]
Hello. Thank you for the prompt response. We appreciate your continued support and efforts!

**Comment by ShayanShamsi** [comment]
Is there a way for teams to keep track of their $10 budget for each of the two models?

**Reply by dedeswim** [reply]
You can use the https://ctf.spylab.ai/api/v1/budget/check API endpoint

**Reply by ShayanShamsi** [reply]
Thank you
